zokrates-worker/src/index.mjs

```diff
@@ -44,7 +44,8 @@ const checkCircuitsOutput = async () => {
       files.map(async f => {
         if (f) {
           const filename = f.split('  ')[1];
-          const circuit = circuits.find(c => filename.includes(c));
+          const _circuit = circuits.find(c => filename.includes(c));
+          const circuit = typeof _circuit === 'undefined' ? '.' : _circuit;
```

Comment:
I don't understand this line... Why do we set `.`?

Reply:
this is my poor attempt at fixing the error that happens in worker when trying to download a file that doesnt exist. It didnt work, but i forgot to remove it. I will remove it as it is useless.